src/widgetastic_patternfly/__init__.py

```diff
@@ -393,6 +393,68 @@ def __repr__(self):
         return '{}({!r})'.format(type(self).__name__, self.locator)
 
 
+class BootstrapNav(Widget):
+    """Encapsulate a Bootstrap nav component"""
+    CURRENTLY_SELECTED = './/li[contains(@class, "active")]/a'
+    TEXT_MATCHING = './/li/a[text()={txt}]'
+    ATTR_MATCHING = './/li/a[@{attr}={txt}]'
+    TEXT_DISABLED = './/li[contains(@class, "disabled")]/a[text()={txt}]'
```

Comment:
text query should be encapsulated in `normalize-space(...)`. I would personally prefer using `.` instead of `text()` as `.` match the text of whole substructure of that element, not just text of that element. Unless there is some other text which would make it difficult :)